tox.ini

```diff
@@ -92,4 +92,4 @@ commands =
 skip_install = true
 description = check code style, e.g. with flake8
 deps = flake8
-commands = flake8 erfa --count --select=E101,W191,W291,W292,W293,W391,E111,E112,E113,E30,E502,E722,E901,E902,E999,F822,F823
+commands = flake8 {toxinidir}/erfa --count --select=E101,W191,W291,W292,W293,W391,E111,E112,E113,E30,E502,E722,E901,E902,E999,F822,F823
```

Comment:
Are you sure this is necessary? I ask because what's there basically is a straight copy of what is in astropy (and hence also in my own affiliated packages, so I care...)

Reply:
Without that I have failures testing locally with:
```
$ tox -e codestyle
codestyle create: /home/antonio/projects/pyerfa/.tox/codestyle
codestyle installdeps: flake8
codestyle installed: appdirs==1.4.3,CacheControl==0.12.6,certifi==2019.11.28,chardet==3.0.4,colorama==0.4.3,contextlib2==0.6.0,distlib==0.3.0,distro==1.4.0,flake8==3.8.4,html5lib==1.0.1,idna==2.8,ipaddr==2.2.0,lockfile==0.12.2,mccabe==0.6.1,msgpack==0.6.2,packaging==20.3,pep517==0.8.2,progress==1.5,pycodestyle==2.6.0,pyflakes==2.2.0,pyparsing==2.4.6,pytoml==0.1.21,requests==2.22.0,retrying==1.3.3,six==1.14.0,urllib3==1.25.8,webencodings==0.5.1
codestyle run-test-pre: PYTHONHASHSEED='1521144942'
codestyle run-test: commands[0] | flake8 erfa --count --select=E101,W191,W291,W292,W293,W391,E111,E112,E113,E30,E502,E722,E901,E902,E999,F822,F823
erfa:0:1: E902 FileNotFoundError: [Errno 2] No such file or directory: 'erfa'
1
ERROR: InvocationError for command /home/antonio/projects/pyerfa/.tox/codestyle/bin/flake8 erfa --count --select=E101,W191,W291,W292,W293,W391,E111,E112,E113,E30,E502,E722,E901,E902,E999,F822,F823 (exited with code 1)
_______________________________________________________ summary ________________________________________________________
ERROR:   codestyle: commands failed
```

Reply:
Yes, I reproduce that, but it doesn't happen on my own package or on astropy, and travis seems happy too. I'll have a look what's different - we might as well try to do the same everywhere...

Reply:
Ah, I see it. In `tox.ini`, in the `[testenv:codestyle]` section, there needs to be the following line:
```
changedir = {toxinidir}
```
See https://github.com/astropy/astropy/blob/dd90339357344a7be5b16e8fc33747ba001b4d16/tox.ini#L147

Reply:
good catch, will to push the fix